src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,89 @@
+import cn from 'classnames';
+import React from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages: number = Math.ceil(total / perPage);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: currentPage === 1,
+          active: currentPage === 1,
```

Comment:
It's not common to have an element be both `active` and `disabled` at the same time. Typically, an `active` class is used to indicate the current page, and `disabled` is used to indicate that an element is not interactive. Please review the logic to ensure that the classes are being applied as intended.